pkg/apis/eksctl.io/v1alpha5/zz_generated.deepcopy.go

```diff
@@ -353,6 +353,11 @@ func (in *NodeGroup) DeepCopyInto(out *NodeGroup) {
 		*out = new(string)
 		**out = **in
 	}
+	if in.VolumeIOPS != nil {
```

Comment:
This should have landed in master, probably my fault when I was rebasing @cPu1's PR...

Reply:
Yep. Maybe the file generation check isn't working as it should?

Reply:
Yes, and it's been fixed in #1034, I think it was to do with the makefile bug you and Gema spotted earlier. Could you please rebase?

Reply:
done